thoth/slo_reporter/configuration.py

```diff
@@ -39,6 +39,7 @@ def __init__(self, start_time: datetime.datetime, end_time: datetime.datetime, n
 
         self.start_time = start_time
         self.end_time = end_time
+        self.last_week_time = str((self.end_time - datetime.timedelta(days=7)).strftime("%Y-%m-%d"))
```

Comment:
Step 0. Here, I find last week datetime, the reason I put it in `configuration`, I want to calculate it only once and only in one place.